Added Few FAQ's (#5555)
harness/developer-hub: kb/platform/harness-platform-faqs.md

#### Can we use an encryption method other than the default with AWS secret manager?

No. When using the AWS secret manager, the default encryption is used. If you have to use a custom encryption, you must use the AWS KMS secret manager. 

> #### Is user can do additional encoding of the harness delegate token?

Reviewer: ```suggestion
#### Can I do additional encoding of the Harness Delegate token?
```

No, the additional encoding is not supported on the delegate token.

> #### In network security policies what harness IP's are need to be whitelisted to connect to the harness?

Reviewer: Please do not use terms like "whitelist" or "blacklist" per industry standard and our style guidelines on inclusive language.

```suggestion
#### In network security policies, what Harness IPs do I need to allowlist to connect to Harness?
```

> You need to allow these IP's: https://developer.harness.io/docs/platform/references/allowlist-harness-domains-and-ips/

Reviewer: Please use human-readable link syntax, such as:

```suggestion
For a list of Harness IPs to allow, go to [Allowlist Harness domains and IPs](https://developer.harness.io/docs/platform/references/allowlist-harness-domains-and-ips).
```
